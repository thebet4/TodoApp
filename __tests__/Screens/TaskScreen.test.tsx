import renderer from 'react-test-renderer';

const TaskScreen = require('../../src/Screens/TasksScreen').default

describe('<TaskScreen />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(TaskScreen).toJSON()
    expect(tree).toMatchSnapshot()
  });
});
