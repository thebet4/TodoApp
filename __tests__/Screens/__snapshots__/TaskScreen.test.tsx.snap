// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TaskScreen /> renders correctly 1`] = `null`;
